src/acpi.rs

```diff
@@ -419,3 +424,357 @@ pub fn process_msct() -> (
         (msc, max_prox_domains)
     }
 }
+
+//TODO: Move this to acpica-sys repo.
```

Comment:
Can we do it?

Reply:
This should be generated automatically after updating the acpica code in https://github.com/gz/acpica-sys. So, we will simply remove this after updating the acpica-sys repo.